### PR TITLE
Auto synchronize directory after drawable import

### PR DESCRIPTION
**Pull request by DVDAndroid**

### CODE REVIEW

**Comment by konifar** [comment]
👀

File: src/main/java/com/konifar/material_icon_generator/MaterialDesignIconGenerateDialog.java [hunk 1]
```diff
@@ -397,18 +399,6 @@ public void actionPerformed(ActionEvent event) {
             }
         });
 
-        comboBoxColor.getAccessibleContext().addPropertyChangeListener(new PropertyChangeListener() {
```
Comment (konifar) [inline src/main/java/com/konifar/material_icon_generator/MaterialDesignIconGenerateDialog.java:400]: Is this okay to remove?
I guess it does not work properly if you remove this.

File: src/main/java/com/konifar/material_icon_generator/MaterialDesignIconGenerateDialog.java [hunk 2]
```diff
@@ -517,6 +507,8 @@ private void create() {
                 JOptionPane.OK_CANCEL_OPTION,
                 JOptionPane.PLAIN_MESSAGE,
                 new ImageIcon(getClass().getResource(ICON_DONE)));
+
+        LocalFileSystem.getInstance().refresh(true);
```
Comment (konifar) [inline src/main/java/com/konifar/material_icon_generator/MaterialDesignIconGenerateDialog.java:522]: Cool! Thanks!

File: src/main/java/com/konifar/material_icon_generator/MaterialDesignIconGenerateDialog.java (hunk 1 above)
Comment (DVDAndroid) [reply src/main/java/com/konifar/material_icon_generator/MaterialDesignIconGenerateDialog.java:400]: @konifar Fixed in latest commit

**Comment by vanniktech** [comment]
This is lovely

**Comment by konifar** [comment]
Looks awesome! Thanks a lot!
I'll release new version soon 😉 
And after released, I'll tell you here.

**Pull request merged**